src/infra/ingest-datafusion/benches/snapshot.rs

```diff
@@ -69,29 +69,29 @@ async fn setup(
     let mut rng = rand::rngs::SmallRng::seed_from_u64(123127986998);
 
     rng.try_fill(&mut buf[..]).unwrap();
-    pk1.append_slice(&buf[orig_range.clone()]);
+    pk1.append_slice(&buf[orig_range]);
     new_pk1.append_slice(&buf[new_range.clone()]);
```

Comment:
`RangeTo` has `Copy` trait 
```rust
#[lang = "RangeTo"]
#[doc(alias = "..")]
#[derive(Copy, Clone, PartialEq, Eq, Hash)]
#[stable(feature = "rust1", since = "1.0.0")]
pub struct RangeTo<Idx> {
    /// The upper bound of the range (exclusive).
    #[stable(feature = "rust1", since = "1.0.0")]
    pub end: Idx,
}
```

`RangeFrom`, by some reason, at the moment nope
```rust
#[lang = "RangeFrom"]
#[doc(alias = "..")]
#[derive(Clone, PartialEq, Eq, Hash)] // not Copy -- see #27186
#[stable(feature = "rust1", since = "1.0.0")]
pub struct RangeFrom<Idx> {
    /// The lower bound of the range (inclusive).
    #[stable(feature = "rust1", since = "1.0.0")]
    pub start: Idx,
}
```

Reply:
I think Rust 2024 edition is planning to do fix `Range` types to make them all `Copy`